tango_simlib/tests/test_dish.py

```diff
@@ -108,14 +111,14 @@ def setUpClassWithCleanup(cls):
         cls.models = tango_sim_generator.configure_device_models(
             cls.data_descr_files, cls.device_name
         )
-        cls.model = cls.models.values()[0]
+        cls.model = list(itervalues(cls.models))[0]
 
     def setUp(self):
         super(test_DishElementMaster, self).setUp()
         self.addCleanup(self._reset_model_defaults)
 
     def _reset_model_defaults(self):
-        for quantity in self.model.sim_quantities.values():
+        for quantity in list(itervalues(self.model.sim_quantities)):
```

Comment:
Isn't safe to just use the .values() in cases like this? In PY2 we would get the list and in PY3 we would get the iterator?

Reply:
This was a defensive change to get the tests to work. following https://python-future.org/compatible_idioms.html#dictionaries

For the iterations I should remove the casting. Thanks.